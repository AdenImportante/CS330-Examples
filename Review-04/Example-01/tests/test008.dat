#! /bin/sh

# Test entire program with only Armour input

cat <<EOF >test008.in
Armour Hat Gold 1 5 Protection 3 water
Armour Boots Diamond 2 4FeatherFalling 4 fire
EOF

# Strip all spaces and newlines
# convert to all lowercase
./storage test008.in 4 | sed 's/[ ]//g' | grep -vE "^$" | tr '[:upper:]' '[:lower:]'