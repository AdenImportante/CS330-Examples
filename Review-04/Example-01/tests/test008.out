processinglog:
(s)hat
(s)boots
playerstoragesummary:
-used50%of4slots
nme:hat
dur:1
def:5
mtl:gold
mdr:protection(lvl3)
emt:water
nme:boots
dur:2
def:4
mtl:diamond
mdr:featherfalling(lvl4)
emt:fire
